gemrb/plugins/OpenALAudio/OpenALAudio.cpp

```diff
@@ -220,12 +230,76 @@ bool OpenALAudioDriver::Init(void)
 	musicThread = SDL_CreateThread( MusicManager, this );
 #endif
 
+	if (!InitEFX()) {
+		Log(MESSAGE, "OpenAL", "EFX not available.");
+	}
+
 	ambim = new AmbientMgrAL;
 	speech.free = true;
 	speech.ambient = false;
 	return true;
 }
 
+bool OpenALAudioDriver::InitEFX(void) {
+	ALCdevice *device = alcGetContextsDevice (alutContext);
+	ALCint auxSends = 0;
+	hasEFX = false;
+
+	/* Would be nice to switch that on and off arbitrarily. */
```

Comment:
this setting is already wired in guiopt, so toggling should work.

Reply:
Ok, I try to move it then.